File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/dto/table/TableResponse.java
package com.hcmute.reservation.dto.table;

import com.hcmute.reservation.model.enums.TableStatus;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class TableResponse {
    private Long tableId;
    private int capacity;
    private TableStatus status;
    private boolean isActive;
    private int version;
}

File: desktop-staff/reservation-winforms/DTO/table/TableResponse.cs
using Newtonsoft.Json;

namespace reservation_winforms.DTO.table
{
    public class TableResponse
    {
        [JsonProperty("tableId")]
        public long TableId { get; set; }

        [JsonProperty("capacity")]
        public int Capacity { get; set; }

        [JsonProperty("status")]
        public string Status { get; set; } // Nhận Enum TableStatus (AVAILABLE, OCCUPIED...)

        [JsonProperty("active")]
        public bool IsActive { get; set; }

        [JsonProperty("version")]
        public int Version { get; set; }
    }
}